src/transformers/models/patchtst/configuration_patchtst.py

```diff
@@ -0,0 +1,262 @@
+            Number of attention heads for each attention layer in the Transformer encoder.
+        share_embedding (`bool`, *optional*, defaults to `True`):
+            Sharing the input embedding across all channels.
+        channel_attention (`bool`, *optional*, defaults to `False`):
+            Activate channel attention block in the Transformer to allow channels to attend each other.
+        ffn_dim (`int`, *optional*, defaults to 512):
+            Dimension of the "intermediate" (often named feed-forward) layer in the Transformer encoder.
+        norm_type (`str` , *optional*, defaults to `"batchnorm"`):
+            Normalization at each Transformer layer. Can be `"batchnorm"` or `"layernorm"`.
+        norm_eps (`float`, *optional*, defaults to 1e-05):
+            A value added to the denominator for numerical stability of normalization.
+        attention_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability for the attention probabilities.
+        dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability for all fully connected layers in the Transformer.
+        positional_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability in the positional embedding layer.
+        path_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout path in the residual block.
+        ff_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability used between the two layers of the feed-forward networks.
+        bias (`bool`, *optional*, defaults to `True`):
+            Whether to add bias in the feed-forward networks.
+        activation_function (`str`, *optional*, defaults to `"gelu"`):
+            The non-linear activation function (string) in the Transformer.`"gelu"` and `"relu"` are supported.
+        pre_norm (`bool`, *optional*, defaults to `True`):
+            Normalization is applied before self-attention if pre_norm is set to `True`. Otherwise, normalization is
+            applied after residual block.
+        positional_encoding_type (`str`, *optional*, defaults to `"sincos"`):
+            Positional encodings. Options `"random"` and `"sincos"` are supported.
+        use_cls_token (`bool`, *optional*, defaults to `False`):
+            Whether cls token is used.
+        init_std (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated normal weight initialization distribution.
+        share_projection (`bool`, *optional*, defaults to `True`):
+            Sharing the projection layer across different channels in the forecast head.
+        scaling (`Union`, *optional*, defaults to `"std"`):
+            Whether to scale the input targets via "mean" scaler, "std" scaler or no scaler if `None`. If `True`, the
+            scaler is set to "mean".
+        mask_input (`bool`, *optional*):
+            Apply masking during the pretraining.
```

Comment:
nit: name to indicate the parameter is a boolean flag rathe than the input itself 

```suggestion
        do_mask_input (`bool`, *optional*):
            Whether or not to apply masking during the pretraining.
```